examples/tools/cmake/cmake_toolchain/use_different_toolchain_generator.rst

```diff
@@ -0,0 +1,79 @@
+
+.. _examples-tools-use-different-toolchain-generator:
+
+Using CMakeToolchain with different generators: Ninja example
+=============================================================
+
+This guide demonstrates how to use :ref:`CMakeToolchain<conan_tools_cmaketoolchain>` with predefined generators
+like `Ninja <https://ninja-build.org/>`_ and how to configure it to use different generators.
+
+Understanding CMake generators
+------------------------------
+
+The `CMake <https://cmake.org/>`_ client offers a variety of `generators <https://cmake.org/cmake/help/latest/manual/cmake-generators.7.html>`_ to create build system files.
+If you want to use a generator other than the default chosen by CMake, you can configure ``tools.cmake.cmaketoolchain:generator``.
+
+Please, note that `CMake <https://cmake.org/>`_ client is not the same as the Conan :ref:`CMake <conan_tools_cmake_helper>` helper.
+
+To see which generators are available on your system, run:
+
+.. code-block:: bash
+
+    $ cmake --help
+
+You can set this :ref:`configuration in your profile <reference_config_files_profiles_conf>`,
+directly in the command line, or even in your :ref:`global configuration<reference_config_files_global_conf_patterns>`.
+
+Using the Ninja generator by default in a profile
+-------------------------------------------------
+
+To set the Ninja generator as the default in a profile file,
+add the entry ``[conf]`` to your profile with the generator value:
```

Comment:
It this is a generic ``myprofile`` then put it a ``caption`` "myprofile", and then use ``conan create . -pr=myprofile`` below.

If this change is proposed to the default profile, say it here: ``add entry to your *default* profile "<cache>/profiles/default" ...``

Reply:
@memsharded I update the example by using a custom profile.